### DISCUSSION TITLE
Failed to connect to the database: [...] host name "nextcloud-aio-database" [...] does not resolve

### DISCUSSION THREAD

**Discussion by Stefomat** [nextcloud/all-in-one#4474] [Questions] [answered]
### Steps to reproduce
1. Let AIO update itself while being on holidays
2. Open Nextcloud Administration Panel
3. See warnings in the log

### Expected behavior 
Nextcloud should be able to successfully connect to the database the first time, or at least wait a bit, before throwing errors.

### Actual behavior 
I'm getting errors in the Nextcloud log whenever i start the containers. These errors appeared on 22nd of March for the first time. This is the time AIO updated to 28.

### Host OS 
Debian 12

#### Nextcloud AIO version 
v8.0.0
#### Current channel 
stable
#### Other valuable info 
![grafik](https://github.com/nextcloud/all-in-one/assets/41241946/351fa25f-c202-42a1-a886-b3bc32756b4b)

**Comment by szaimen** [comment, marked as answer]
Hi, I fear I cannot reproduce this. In my testing, the nextcloud container waits for database container to start before starting itself. However you might ne running into this: https://github.com/nextcloud/all-in-one/discussions/2557

**Reply by Stefomat** [reply]
Unfortunately it can't be that, I haven't activated IPv6.
